src/axom/primal/tests/primal_polyhedron.cpp

```diff
@@ -284,24 +310,21 @@ void check_volume()
 {
   const int DIM = 3;
   using PolyhedronType = primal::Polyhedron<double, DIM>;
+  using PointType = primal::Point<double, 3>;
 
-  umpire::ResourceManager& rm = umpire::ResourceManager::getInstance();
-
-  // Save current/default allocator
-  const int current_allocator = axom::getDefaultAllocatorID();
-
-  // Determine new allocator (for CUDA or HIP policy, set to Device)
-  umpire::Allocator allocator =
-    rm.getAllocator(axom::execution_space<ExecSpace>::allocatorID());
+  constexpr double EPS = 1e-4;
```

Comment:
Is it possible to lower this `EPS` or was there some imprecision in the calculation that requires it to be so high?